application.tf

```diff
@@ -418,17 +418,21 @@ module "secrets-operator-setup" {
     time_sleep.wait_1_minutes_after_cluster,
     kubernetes_secret.harbor-pull-secret
   ]
-  count           = var.secrets_operator_enabled == true && var.multitenant_enabled == false ? 1 : 0
-  source          = "./modules/common/vault-secrets-operator-setup"
-  vault_address   = var.vault_address
-  account         = var.aws_account
-  region          = var.region
-  name            = var.label
-  kubernetes_host = local.environment_cluster_kubernetes_host
-  vault_username  = var.vault_username
-  vault_password  = var.vault_password
-  audience        = ""
-  environment     = var.load_environment == "" ? local.environment : lower(var.load_environment)
+  count                                  = var.secrets_operator_enabled == true && var.multitenant_enabled == false ? 1 : 0
+  source                                 = "./modules/common/vault-secrets-operator-setup"
+  vault_address                          = var.vault_address
+  account                                = var.aws_account
+  region                                 = var.region
+  name                                   = var.label
+  kubernetes_host                        = local.environment_cluster_kubernetes_host
+  vault_username                         = var.vault_username
+  vault_password                         = var.vault_password
+  audience                               = ""
+  environment                            = var.load_environment == "" ? local.environment : lower(var.load_environment)
+  lambda_sns_forwarder_enabled           = var.lambda_sns_forwarder_enabled
+  lambda_sns_forwarder_iam_principal_arn = local.environment_lambda_sns_forwarder_iam_principal_arn
+  aws_access_key                         = var.aws_access_key
```

Comment:
Why are these being passed in?

Reply:
vault aws integration requires to setup aws credentials for the account